webapp/app/.server/db/index.ts

```diff
@@ -15,3 +16,20 @@ export const db = drizzle(queryClient, {
     },
   },
 });
+
+async function testDbConnection() {
+  try {
+    await db.select().from(proposalsTable).limit(1);
+
+    console.log("Connected to the database successfully");
+  } catch (error) {
+    if (error instanceof Error) {
+      console.error("An error occurred while connecting to the database:", error.name);
+      console.error(error.message);
+      console.error(error.cause);
+    }
+    throw new Error("Failed to connect to the database");
+  }
+}
+
+testDbConnection();
```

Comment:
shouldn't this be awaited to make the error stop the app if connection fails?

Reply:
Done

Reply:
Refactored, now it's in a separate start up routine for the express server